content/installation/Teams/_index.md

```diff
@@ -24,60 +24,86 @@ We will be discussing the most common way i.e exposing using [ingress](https://k
+| Developer Information / Developer ... | BotKube                                                                                                                        |
+| Developer Information / Website       | https://botkube.io                                                                                                             |
+| App URLs / Privacy statement          | https://botkube.io/privacy                                                                                                     |
+| App URLs / Terms of use               | https://botkube.io/license                                                                                                     |
+| Application (client) ID               | Add the Application (client) ID you obtained from Azure Active Directory                                                       |
 
-1. Click "Set up"
+   ![](/images/teams_add_app_info.png "Teams add app info")
 
-2. Set "BotKube" name, and enable
-    - Messaging bot: **[x] My bot supports uploading and downloading files**
-    - Scope: **[x] Personal** & **[x] Team**
-    And Create bot.
+6. Click the "Save" button to save your details.
 
-{{% notice note %}}
-You can also use a custom username for your bot. However, it needs to be passed during BotKube installation in one of the further steps.
-{{% /notice%}}
+7. Navigate to "Configure / Branding" left-hand side menu item. Click to open the "Branding" section.
 
-3. On the "Bots" page, click on **Generate a new password**. Note down the password required while installing BotKube backend.
+8. Download BotKube icons from https://github.com/kubeshop/botkube/tree/main/branding/logos and update Branding icons.
 
-4. **Copy App ID displayed below Bot name in the heading. Note that Bot App ID is different than the one we generate on the "Apps Details" page.**
+####  Add the Bot feature to the App
 
-5. Set **Messaging Endpoint**.
-   The messaging endpoint is the URL on which BotKube backend listens for incoming requests from Teams. While deploying the BotKube backend you can give an option to expose BotKube via Ingress. Please check the [prerequisites](/installation/teams/#prerequisites) for more details.
+On the left-hand side menu click "Configure / App features"
 
-   ![](/images/teams_bot.png "Teams BotKube Bot")
+1. In "App features / Select a feature to add", click the "Bot" button
+
+   ![](/images/teams_add_bot_feature.png "Teams add bot feature to app")
+
+2. In "Bot / Identify your bot" enable
+    - What can your bot do?: **[x] Upload and download files**
+    - Select the scopes in which people can use this command: **[x] Personal** & **[x] Team**
+
+3. Then click "Save".
+
+4. In "Bot / Identify your bot", select "Create a bot", click the "+ New Bot" and enter a name for the Bot. Use this name when BotKube's MS Teams `botName` later.
```

Comment:
This comment is related to setting up the botName when running helm. You have to name the bot with the name you assigned it here, in your case `TRBotKube`.

It could def be clarified. 👍